src/common/obj_bencher.cc

```diff
@@ -17,6 +17,7 @@
  */
 #include <pthread.h>
 #include "common/Cond.h"
+#include "include/compat.h"
```

Comment:
On Sat, May 28, 2016 at 8:28 PM, Willem Jan Withagen <
notifications@github.com> wrote:

> In src/common/obj_bencher.cc
> <https://github.com/ceph/ceph/pull/9371#discussion_r64988367>:
>
> > @@ -17,6 +17,7 @@
> >   */
> >  #include <pthread.h>
> >  #include "common/Cond.h"
> > +#include "include/compat.h"
>
> @ukernel <https://github.com/ukernel> I'd suggest putting compat.h as
> very first include file.
> This will prevent other include files from defining things that need to be
> repaired later.
> Like ENODATA
>

there are some "#ifndef; #define xxx; #endif". I think putting compat.h at
first causes problem


> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph/pull/9371/files/d63fe5b93266824212264712ff71ca09968d273e..7425afb261d17a8e1d75c47c63b29739fd5d4f8c#r64988367>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACpZr23vNKKcY8AsDrCKEZjJ5Ky9yUAJks5qGDTugaJpZM4IokCx>
> .
>